src/utils/map-state.js

```diff
@@ -174,11 +174,7 @@ export default class MapState {
    * @param {Number} deltaScaleY - a number between [-1, 1] specifying the
    *   change to pitch. -1 sets to minPitch and 1 sets to maxPitch.
    */
-  rotate({deltaScaleX, deltaScaleY}) {
-    assert(deltaScaleX >= -1 && deltaScaleX <= 1,
```

Comment:
Assume we are removing these asserts, without replacing them with any `ensureFinite`s since these values are typically not supplied by the user. Makes sense.

Reply:
I added default values for the params:
`rotate({deltaScaleX = 0, deltaScaleY = 0})`
So `ensureFinite` won't be needed.

Reply:
> So ensureFinite won't be needed.

OK should be good enough. As far as I know, default params only trigger when the field is completely omitted. The user can still send in bad params. Calling this with `({deltaScaleX: undefined})`, `({deltaScaleX: null})` etc would still cause problems.

Reply:
I believe `{deltaScaleX: undefined}` will still work but indeed `{deltaScaleX: null}` breaks.

Reply:
Yes, you are correct.